blazar_dashboard/static/leases/js/calendar/lease_chart.js

```diff
@@ -59,6 +59,20 @@
     selector = '#blazar-calendar-network'
     pluralResourceType = gettext("Networks")
     rowAttr = "segment_id";
+    
+    chooserAttr = "network_type";
+    chooserAttrPretty = gettext("Network Type");
+    populateChooser = function (chooser, availableResourceTypes) {
+      const networkTypesPretty = [
+          ['vlan', gettext("VLAN")],
```

Comment:
I forget how this works, is this displaying the text "VLAN", but filtering based on the first key "vlan"? I'm wondering if it makes more sense to omit the chooser entirely for networks (just hiding it instead of populating it), or to base it on "usage_type" or "stitch_provider", as those will be what a user cares about. Those might have implications on associate sites though, as they might not have either of those keys set. This leads me to think that we should omit the chooser until we figure out the best solution.